universidade-estadual-do-oeste-do-parana-programa-institucional-de-bolsas-de-iniciacao-cientifica.csl

```diff
@@ -58,7 +93,7 @@
     <choose>
       <if type="chapter" match="any">
         <text term="in" text-case="capitalize-first" prefix=" " suffix=" "/>
-        <names variable="editor translator" delimiter=", " suffix=" ">
+        <names variable="editor translator" delimiter=", " prefix=" " suffix=" ">
```

Comment:
The new prefix (`prefix=" "`) here doesn't seem necessary, since the preceding `<text/>` element already has `suffix=" "`. Can you delete it again?